### PR TITLE
FullyPacks Redesign

### PR DESCRIPTION
**Pull request by JeremiahHerring**
Merge FullyPacks Changes into Main

### CODE REVIEW

**Comment by JeremiahHerring** [comment]
image
I made the FullyPacks look like the Figma design

**Comment by tomasohCHOM** [comment]
Can we rename `.fullypacks-title` in global.css to `.feed-title`, we could reuse this class in different section titles for the Feed page.

File: package.json [hunk 1]
```diff
@@ -41,6 +40,7 @@
     "@types/react-timeago": "^4.1.3",
     "autoprefixer": "^10.4.13",
     "postcss": "^8.4.21",
+    "prisma": "^5.9.0",
```
Comment (anhduy1202) [inline package.json:43]: can you revert it to the original version 4.11.0, just for consistency?

Comment (JeremiahHerring) [reply package.json:43]: oh yeah I thought I changed that I guess I didn't commit it

**Pull request merged**